src/main/scala/com/kinja/amqp/persistence/InMemoryMessageBufferDecorator.scala

```diff
@@ -36,6 +36,10 @@ class InMemoryMessageBufferDecorator(
 
 	logger.debug("Memory flusher scheduled")
 
+	override def hasMessageToProcess(): Future[Boolean] = {
+		messageStore.hasMessageToProcess()
```

Comment:
So, if `rabbitmq_messages` table is empty, we do not process messages that are in memory either? Doesn't seem right.

Reply:
Off-line conversation: that's what happens anyway.